frontend/settingsTab.js

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { Button, View, Text, Switch } from 'react-native';
+import { styles } from './style';
+import { createStackNavigator } from '@react-navigation/stack';
+
+function Settings({ navigation }) {
+    const [ darkMode, setDarkMode ] = React.useState(false);
+
+    return (
+        <View style={[styles.settings, {
+            backgroundColor: darkMode ? '#000' : '#fff',
+        }]}>
+            <Button onPress={() => navigation.navigate('Preferences')} title='Preferences' />
+            <Button onPress={() => navigation.navigate('Profile')} title='Profile' />
+            <View style={styles.darkMode}>
+                <Text style={[styles.darkModeText, {
+                    color: darkMode ? '#fff' : '#000'
+                }]}>Dark Mode</Text>
+                <View style={styles.darkModeToggleView}>
+                    <Switch onChange={() => setDarkMode(!darkMode)} value={darkMode} />
+                </View>
+            </View>
+            <Button onPress={() => navigation.navigate('Notifications')} title='Notifications' />
+            <Button onPress={() => navigation.navigate('Time Zone')} title='Time Zone' />
+        </View>
+    );
```

Comment:
A mini implementation of dark mode. However, this only works on the Settings page and is currently a bug as per [Issue #9 ](https://github.com/williamchong256/immersive-sleep/issues/9)